src/doc/rustc/src/platform-support/wasm32-unknown-emscripten.md

```diff
@@ -0,0 +1,157 @@
+# `wasm32-unknown-emscipten`
+
+**Tier: 2**
+
+The `wasm32-unknown-emscripten` target is a WebAssembly compilation target which
+uses the [Emscripten](https://emscripten.org/) compiler toolchain. Emscripten is
+a C/C++ toolchain designed to make it as easy as possible to port C/C++ code
+written for Linux to run on the web. It thus provides POSIX-compatible (musl) `libc`
+and `libstd` implementations and many Linux APIs, access to the OpenGL and SDL APIs,
+and the ability to run arbitrary JavaScript code, all based on web APIs using JS
+glue code. With the `wasm32-unknown-emscripten` target, Rust code can interoperate
+with Emscripten's ecosystem, C/C++ and JS code, and web APIs.
```

Comment:
It would be nice if we could mention somewhere in here that emscripten code also can run in node or other JavaScript runtimes. (The goal of Emscripten is to run on web, but once you run on web it's not too hard to also run in other JS runtimes.)

Reply:
Done